meson.build

```diff
@@ -12,39 +12,25 @@ parson_sources = ['parson.c']
 
 parson_inc = include_directories('.')
 
-lib_so_version = '0'
-
-parson_shared_lib = shared_library(
-    meson.project_name(),
-    sources: parson_sources,
-    soversion: lib_so_version,
-    install: true
-)
-
-parson_static_lib = static_library(
+parson_lib = library(
     meson.project_name(),
     sources: parson_sources,
     install: true
 )
```

Comment:
library(), like shared_library(), accepts the soversion argument -- it is used only when building the shared component. Was this intentionally dropped?

Aside: just noticed in commit da126c2abafa061cc7421168c05b9417d63b63b1 the project version was added as the library soversion.

But in https://github.com/kgabis/parson/issues/179#issuecomment-1081142821 it was predicted that there shouldn't really be breaking ABI changes between versions... and the soversion was set to "0" here.

Which one is it? (The cmake one seems wrong to me, since it says the ABI is broken every time PARSON_VERSION is updated.)

Reply:
Yes, SOVERSION in cmake doesn't seem to make sense but at this point I don't know if removing it might break something.

I don't use or know meson and cmake so I'm open to PRs fixing those issues.